lookout/style/format/analyzer.py

```diff
@@ -223,7 +223,10 @@ def train(cls, ptr: ReferencePointer, config: Mapping[str, Any], data_service: D
                 "\n\t".join("%-55s %.5E" % (fe.feature_names[i], importances[i])
                             for i in numpy.argsort(-importances)[:25] if importances[i] > 1e-5))
             # throw away imprecise classes
-            model[language] = trainable_rules.rules
+            if trainable_rules.rules.rules:
+                model[language] = trainable_rules.rules
+            else:
+                cls._log.warning("model for %s had 0 rule. Skipping" % language)
```

Comment:
```suggestion
                cls._log.warning("Model have 0 rule for %s. Skipping." % language)
```
?

Reply:
we have really inconsistent style so I think it's really arbitrary to pick one or the other (the 2 styles seem to be 50/50 in analyzer.py). We could do a pass to harmonize at some point though